corehq/motech/fhir/views.py

```diff
@@ -44,8 +46,9 @@ def page_url(self):
 
 
 @require_GET
-@login_or_api_key
-@require_superuser
+@api_key_auth
+@require_can_edit_data
```

Comment:
@kaapstorm checking that `user_can_access_case` would still be needed along with this decorator to make it location safe?